apps/L1E/ticket-booking/ticket-booking-frontend/specs/components/DataPickerAdmin.test.tsx

```
@@ -0,0 +1,56 @@
+/**
```

ESLint

[error] Filename is not in pascal case. Rename it to `DataPickerAdmin.Test.tsx`.

(unicorn/filename-case)